RELICENSING.md

```diff
@@ -138,6 +138,8 @@ the Developer Certificate of Origin, version 1.1:
 - Carl Rand (crand)
 - Gregor Mückl (gmueckl)
 
+...
```

Comment:
all of my contributions originally submitted to this project under the 3-Clause BSD License are hereby relicensed to the Apache License, Version 2.0, and are submitted pursuant to the Developer Certificate of Origin, version 1.1